Shared/Account/AccountModel.swift

```diff
@@ -25,6 +26,11 @@ extension AccountError: LocalizedError {
                 "Username not found. Did you use your email address by mistake?",
                 comment: ""
             )
+        case .invalidServerURL:
+            return NSLocalizedString(
+                "The server entered doesn't appear to be a valid URL. Please check what you've entered and try again.",
```

Comment:
Thinking of users who are uninitiated, I think we should include some example values in this error message instead of simply saying something is wrong. Perhaps verbiage like this?

> Please enter a valid instance domain name. It should look like "https://example.com" or "write.as".

Reply:
Sure!

<img src="https://user-images.githubusercontent.com/387655/106631914-3751ea80-654b-11eb-8e0c-8840aa87578f.png" width="375" />

Reply:
Looks good!